src/types.tsx

```diff
@@ -0,0 +1,4 @@
+export interface data {
```

Comment:
This looks like a template for exporting your data
Does this data come from your store?

Reply:
This is an interface/type declaration for typescript. This is just a placeholder until we define it.

Reply:
Very interesting! Thanks for the information!